src/components/Square.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './Square.css'
+
+const Square = (props) => {
+  // For Wave 1 enable this 
+  //  Component to alert a parent 
+  //  component when it's clicked on.
+
+  return <button
+    className="square"
+
+  >
```

Comment:
I suspect you're leaving room for additional class names but it makes this hard to parse:

```suggestion
    className="square"
  >
```